<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>856</width>
    <height>925</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>H5 View and Plot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>File</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="data_filename_lineEdit"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>File:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="data_filename_browse_pushButton">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="4">
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="dataview_page">
           <attribute name="title">
            <string>Data View</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="dataview_placeholder">
              <property name="text">
               <string>Once a file is loaded, data will show up here.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="plot_page">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <attribute name="title">
            <string>Plot</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Datasets</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <widget class="QListWidget" name="dataset_listWidget"/>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Plot data</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="0" colspan="2">
                <widget class="QStackedWidget" name="data_stackedWidget">
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="dataplot_page">
                  <layout class="QGridLayout" name="gridLayout_6">
                   <item row="0" column="0">
                    <widget class="PlotWidget" name="data_plotWidget"/>
                   </item>
                   <item row="1" column="0">
                    <layout class="QGridLayout" name="gridLayout">
                     <item row="0" column="1">
                      <widget class="QDoubleSpinBox" name="plotWidget_x_start_spinBox">
                       <property name="maximumSize">
                        <size>
                         <width>203</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="maximum">
                        <double>9999999.000000000000000</double>
                       </property>
                       <property name="value">
                        <double>0.000000000000000</double>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>X start</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="3">
                      <widget class="QDoubleSpinBox" name="plotWidget_x_end_spinBox">
                       <property name="maximum">
                        <double>9999999.000000000000000</double>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="2">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>X end</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                 <widget class="QWidget" name="imageItem_page">
                  <layout class="QGridLayout" name="gridLayout_7"/>
                 </widget>
                 <widget class="QWidget" name="imageView_page">
                  <layout class="QGridLayout" name="gridLayout_8">
                   <item row="0" column="0" colspan="2">
                    <widget class="ImageView" name="data_imageView"/>
                   </item>
                   <item row="1" column="0" colspan="2">
                    <layout class="QGridLayout" name="gridLayout_11">
                     <item row="0" column="3">
                      <widget class="QDoubleSpinBox" name="imageView_x_end_spinBox">
                       <property name="maximum">
                        <double>9999999.000000000000000</double>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QDoubleSpinBox" name="imageView_x_start_spinBox">
                       <property name="maximumSize">
                        <size>
                         <width>203</width>
                         <height>16777215</height>
                        </size>
                       </property>
                       <property name="maximum">
                        <double>9999999.000000000000000</double>
                       </property>
                       <property name="value">
                        <double>0.000000000000000</double>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="0">
                      <widget class="QLabel" name="label">
                       <property name="text">
                        <string>X start</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="2">
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>X end</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
               <item row="0" column="0">
                <widget class="QPushButton" name="plot_pushButton">
                 <property name="text">
                  <string>Plot</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QGroupBox" name="plot_type_groupBox">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="title">
                  <string>Type</string>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QRadioButton" name="plot_radioButton">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="text">
                     <string>Plot</string>
                    </property>
                    <property name="checked">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QRadioButton" name="image_radioButton">
                    <property name="enabled">
                     <bool>true</bool>
                    </property>
                    <property name="text">
                     <string>Image</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>856</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
